src/brainglobe_napari/tests/test_atlas_viewer_widget.py

```diff
@@ -0,0 +1,44 @@
+from typing import Tuple
+
+import pytest
+from napari.viewer import Viewer
+
+from brainglobe_napari.atlas_viewer_widget import AtlasViewerWidget
+
+
+@pytest.fixture
+def make_atlas_viewer(make_napari_viewer) -> Tuple[Viewer, AtlasViewerWidget]:
+    """Fixture to expose the atlas viewer widget to tests.
+    Simultaneously acts as a smoke test that the widget can
+    be instantiated without crashing."""
+    viewer = make_napari_viewer()
+    atlas_viewer = AtlasViewerWidget(viewer)
+    return viewer, atlas_viewer
+
+
+@pytest.mark.parametrize(
+    "row,expected_atlas_name",
+    [
+        (0, "example_mouse_100um"),
```

Comment:
I would use some smaller atlases here. Otherwise the tests will take ages on new machines.

Reply:
We can additionally cache some of them on GH actions #13

Reply:
True, but these would be slow even just running on a new contributors machine. `allen_mouse_10um` is 7GB.